tests/rustdoc-ui/disambiguator-endswith-named-suffix.rs

```diff
@@ -0,0 +1,110 @@
+//@ check-pass
```

Comment:
You need to add:

```
//@ normalize-stderr-test: "nightly|beta|1\.[0-9][0-9]\.[0-9]" -> "$$CHANNEL"
```

And then to remove and regenerate stderr files (for both tests).

Reply:
Is it necessary to add this header for [disambiguator-macro-endswith-exclamatory](https://github.com/rust-lang/rust/pull/127016/files#diff-e34fcf12f4082ceb0e0fe60cdd6522cf18abac1a11b1a79c34633a6f93a1388f), I'm not sure (because I haven't seen any related snapshots in it)

Reply:
Like they said. :)